package.json

```diff
@@ -225,22 +225,30 @@
     "cover-integration": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/integration/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
     "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
-    "----------------------------------------- local test dbs ------------------------------------------": "",
+    "----------------------------------------- bulk tests ------------------------------------------": "",
+    "start-all": "grep 'bash dev' package.json | grep start | grep -vP \"(all)\" | cut -d'\"' -f4 | sort | xargs -n1 | grep dev | xargs -n1 bash",
+    "test-all": "grep 'test-' package.json | grep -vP \"(unit|integration|smoke|typings)\" | cut -d'\"' -f4 | xargs -n4 yarn ",
```

Comment:
I'm also not a fan of using grep to find the different commands. I feel it could eventually clash with something similarly named in the file.

Even though it's verbose, I prefer the `test-unit-all` approach.
You could add a `test-integration-all` that is written similarly to `test-unit-all`, then this command can be `yarn test-unit-all && yarn test-integration-all && yarn test-typings && yarn lint-no-fix`

Reply:
Yeah I have my concerns. I weighed the frequency the dev containers would be upgraded against the potential collision.  If moving to a separate script, It'd be better to iterate over supported dialects.